socorro/external/es/super_search_fields.py

```diff
@@ -92,9 +92,10 @@ def create_field(self, **kwargs):
             )
         except elasticsearch.exceptions.ConflictError:
             # This field exists in the database, it thus cannot be created!
-            raise InsertionError(
-                'The field "%s" already exists in the database, '
-                'impossible to create it. ' % params['name']
+            raise BadArgumentError(
+                'name',
+                msg='The field "%s" already exists in the database, '
+                    'impossible to create it. ' % params['name'],
```

Comment:
I would recommend using `%r` instead if you ever have a need to use `"%s"`. E.g. 
```
msg='The field %r already ...'
```